src/get.js

```diff
@@ -1,59 +1,92 @@
-import { isString, isArray, isFunction } from './util';
+import { isString, isArray, isFunction } from "./util";
 
 let resolved = {};
 
 /**
  * Load javascript url asynchronously.
  *
  * @param {*} url               The javascript url to load.
+ * @param {*} type              Url type 'text/css or text/javascript'
  * @param {*} callback          Callback function to invoke on success.
  * @param {*} errorCallback     Callback function to invoke on error.
  * @returns {void}
  */
-function loadScript(url, callback, errorCallback) {
-  let invokeCallback = function() {
+function loadScript(
+  url,
+  type = "text/javascript",
```

Comment:
use var,let or const declaration.

Reply:
the type is for default value assign in function, if no other parameters are given and only url is provided in loadScript function. 

function loadScript(
  url,
  type = "text/javascript"){
}